Makefile

```diff
@@ -33,6 +33,7 @@ deploy_run:
 		-e PROJECT_DIR=$(PWD) \
 		-e HOST_UID=$(UID) \
 		-e HOST_GID=$(GID) \
+		-e RUNTIME_ENV=$(RUNTIME_ENV) \
```

Comment:
Can we document where and how we intend for this to be set in our README or as a comment?